pkgs/tools/networking/globalprotect-openconnect/default.nix

```diff
@@ -1,22 +1,19 @@
 { stdenv, lib, fetchFromGitHub
-, cmake, qtwebsockets, qtwebengine, wrapQtAppsHook, openconnect
+, cmake, qtwebsockets, qtwebengine, qtkeychain, wrapQtAppsHook, openconnect
 }:
 
 stdenv.mkDerivation rec {
   pname = "globalprotect-openconnect";
-  version = "1.4.8";
+  version = "1.4.9";
 
-  src = fetchFromGitHub {
-    owner = "yuezk";
-    repo = "GlobalProtect-openconnect";
-    fetchSubmodules = true;
-    rev = "v${version}";
-    sha256 = "sha256-PQAlGeHVayImKalCNv2SwPcxD0ts4BVSqeo1hKYmnMA=";
+  src = fetchurl {
+    url = "https://github.com/yuezk/GlobalProtect-openconnect/releases/download/v${version}/globalprotect-openconnect-${version}.tar.gz";
+    sha256 = "0b7s3gf0gznlpmf9dxfq77254zlxmpajhzzn3scrdrvf413sjl0f";
```

Comment:
This may also need to be updated.

Reply:
Also, what's the reason for using `fetchurl` instead?

Reply:
I failed to fetch [the new submodule](https://github.com/yuezk/GlobalProtect-openconnect/blob/8ca261055096c3a740b096a8d8106028bb82d76f/.gitmodules#L10) in my NixOS, may related to https://github.com/NixOS/nixpkgs/issues/195117